var/spack/repos/builtin/packages/freeglut/package.py

```diff
@@ -27,6 +27,7 @@ class Freeglut(CMakePackage, SourceforgePackage):
     depends_on('inputproto')
 
     patch('common-gcc10.patch', when="@3.2.1: %gcc@10.0:")
+    patch('common_patch_clang.patch', when="@3.2.1: %clang@11.0:")
```

Comment:
Can we:
 1. Retrieve the patch from some `url` instead of storing it in Spack?
 2. Check if this patch also works for GCC 10 and later (and in case remove the other patch)?

Also, it would be nice to have a brief comment on top of the patch directive that explains what the patch does (for better readability of the package recipe).

Reply:
Sure, I just checked that the clang patch also works with gcc 11.